build/tools.go

```diff
@@ -33,4 +33,7 @@ import (
 
 	// submodule test dependencies
 	_ "github.com/armon/go-socks5" // for staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper_test.go
+
+	// license dependencies (until https://github.com/kubernetes/kubernetes/issues/113366 is addressed)
```

Comment:
this shouldn't be needed ... this looks like a flaw in our license verification script, not something we should add a direct dependency for

Reply:
Indeed, I’ll look into fixing that separately and rework this PR on top of that.